pleskdistup/common/src/action.py

```diff
@@ -390,8 +390,8 @@ def __exit__(self, *kwargs):
 
     def _get_flow(self) -> typing.Dict[str, typing.List[ActiveAction]]:
         res = {}
-        for stage_id, actions in reversed(self.stages.items()):
-            res[stage_id] = list(reversed(actions))
+        for stage_id, actions in reversed(list(self.stages.items())):
```

Comment:
It's a temporary iterator, why use `list` here?

Reply:
Because it seems like python 3.6 doesn't support `reversed` for dict_items. Described in the commit message